### PR TITLE
Improve content on sign in page

### PR DESCRIPTION
**Pull request by SteveMilnes**
## Changes

This work adds more content to the sign in page so that it functions like a service start page.

localhost_3000_sign-in

## Checklist

- [x] Attach this pull request to the appropriate card in Trello.
- [x] Update the `CHANGELOG.md` if needed.

### CODE REVIEW

File: app/views/sessions/new.html.erb [hunk 1]
```diff
@@ -1,7 +1,22 @@
...
   <%= button_tag(t("sign_in.button", type: :submit), class: "govuk-button") %>
 <% end %>
+<h2 class="govuk-heading-m">Register for access</h2>
+<p class="govuk-body">If you do not have access you can <a href="https://forms.office.com/Pages/ResponsePage.aspx?id=yXfS-grGoU2187O4s0qC-X7F89QcWu5CjlJXwF0TVktUMEpZSU5aTEVYVlIyRE1JSjdMREtIVDhWQiQlQCN0PWcu" class="govuk-link" rel="noreferrer noopener" target="_blank">sign up to use Complete conversions, transfers and changes (opens in new tab)</a>.</p>
+<p class="govuk-body">To register, you’ll need to provide your:</p>
+<ul class="govuk-list govuk-list--bullet">
+  <li>full name</li>
+  <li>@‌education.gov.uk email address</li>
+  <li>team information</li>
+  <li>job title</li>
```
Comment (mec) [inline app/views/sessions/new.html.erb:16]: Does the form ask for this? Is that how we work out the if they are a 'lead'?

**Review by mec — CHANGES_REQUESTED** [review]
Nice work getting this!! Looks 1000x better now and will be useful for out users! I left a couple of comments.

File: app/views/sessions/new.html.erb [hunk 2]
```diff
@@ -1,7 +1,22 @@
 <h1 class="govuk-heading-xl">
   Sign in
 </h1>
-
+<p class="govuk-body">Use this product if you’re a delivery officer or caseworker at the Department for Education.</p>
```
[the remark this answers is not in the document]
Comment (SteveMilnes) [reply app/views/sessions/new.html.erb:4]: Yep, good point. I'll update that.

File: app/views/sessions/new.html.erb (hunk 1 above)
Comment (SteveMilnes) [reply app/views/sessions/new.html.erb:16]: Yeah, it asks your name, email, which team you're in and what your job title is (or grade where folks just get called the G6 for a particular region for example). Once we know that we can work out if they're a delivery officer, caseworker or RCS team lead/somebody who needs to assign work.

File: app/views/sessions/new.html.erb [hunk 3]
```diff
@@ -1,7 +1,22 @@
...
+<p class="govuk-body">You can use this product to manage work that helps DfE to convert schools to academies.</p>
```
[the remark this answers is not in the document]
Comment (SteveMilnes) [reply app/views/sessions/new.html.erb:5]: I've removed the govuk-body stuff.

**Review by mec — APPROVED** [review]

**Pull request merged**